package.json

```diff
@@ -23,6 +23,9 @@
     "cypress:test": "npm run cypress:clean && wait-on tcp:8000 && cypress run --spec ./cypress/integration/endpoint-tests.js",
     "cypress:test:a11y": "npm run cypress:clean && wait-on tcp:8000 && cypress run --spec ./cypress/integration/a11y-tests.js",
     "cypress:test:manual": "cypress open",
+    "fix": "run-s fix:*",
+    "fix:js": "prettier --write \"cypress/integration/*.js\" \"source/js/**/*.js\" \"source/js/**/*.jsx\" webpack.config.js",
```

Comment:
Is there an advantage of using `prettier --write` over `eslint --fix` and `stylelint --fix`? If not, I think we should just copy the `test:eslint:js` task and `test:scss` task and add the `--fix` option to them for these new `fix` tasks. This way we can make sure we use the same config to lint and to fix.

Reply:
I'm also wondering if fixing linting issues should be a separate step: what would be the issue with adding `--fix` to the existing tests commands?

Reply:
the prettier folks warn to never use `--fix` by default because it _can_ break things. So I really want to keep this a separate task.